examples/with-apollo/lib/exenv.js

```diff
@@ -0,0 +1,2 @@
+export const IS_SERVER = typeof window === 'undefined'
+export const IS_BROWSER = !IS_SERVER
```

Comment:
Actually as shown in https://github.com/possibilities/next-with-auth, you can simplify just using `process.browser`.